lib/oxidized/input/ftp.rb

```diff
@@ -32,15 +28,6 @@ def cmd file
       @ftp.getbinaryfile file, nil
     end
 
-    # meh not sure if this is the best way, but perhaps better than not implementing send
```

Comment:
How come the rest of this code block was removed?

Reply:
a) to mirrored tftp.rb
b) The send my_proc define isn't called anywhere and define output was removed as the output log was empty and could have been related. Still empty but its removal hasn't broken anything